firefox-ios/firefox-ios-tests/Tests/ClientTests/Toolbar/AddressBarStateTests.swift

```diff
@@ -428,35 +428,48 @@ final class AddressBarStateTests: XCTestCase, StoreTestUtility {
         XCTAssertFalse(newState.showQRPageAction)
     }
 
-    func test_cancelEditAction_onHomepage_returnsExpectedState() {
+    func test_cancelEditOnHomepageAction_withURL_returnsExpectedState() {
         setupStore()
         let initialState = createSubject()
         let reducer = addressBarReducer()
+        let didChangeURLAction = ToolbarAction(url: URL(string: "https://mozilla.com")!,
+                                               windowUUID: windowUUID,
+                                               actionType: ToolbarActionType.urlDidChange
+        )
+        let stateWithURL = reducer(
+            initialState,
+            didChangeURLAction
+        )
```

Comment:
nit: seems short enough to live in one line, decrease line length in the class and more readable in my opinion
```suggestion
        let stateWithURL = reducer(initialState, didChangeURLAction)
```